app/bundles/LeadBundle/Controller/LeadController.php

```diff
@@ -1354,6 +1354,12 @@ public function emailAction($objectId = 0)
 
                         $mailer->setSubject($email['subject']);
 
+                        /** @var EmailModel $emailModel */
+                        $emailModel = $this->getModel('email');
+                        if (($emailEntity = $emailModel->getEntity((int) $email['templates']))) {
+                            $mailer->setEmail($emailEntity);
+                        }
+
```

Comment:
Should move this code 

```
// Set Content
$mailer->setBody($email['body']);
$mailer->parsePlainText($email['body']);
```

after `$mailer->setEmail($emailEntity);` condition

This prevent reset changed content form email entity